docs/notifications/getting-started.md

```diff
@@ -0,0 +1,220 @@
+---
+id: getting-started
+title: Getting Started
+description: How to get started with the notifications and signals
+---
+
+The Backstage Notifications System provides a way for plugins and external services to send notifications to Backstage users.
+These notifications are displayed in the dedicated page of the Backstage frontend UI or by frontend plugins per specific scenarios.
+Additionally, plugins can implement processors to send notifications through external channels like email, Slack, or MS Teams.
+
+Notifications can be optionally integrated with the signals (a push mechanism) to ensure users receive them immediately.
```

Comment:
```suggestion
Notifications can be optionally integrated with the Signals API (a push mechanism) to ensure users receive them immediately.
```

Or is it more a plugin? API feels more correct but I could be wrong.

Reply:
It referes rather to the plugin. If it is deployed, then it is automatically used.